docs/analyitics.md

```diff
@@ -0,0 +1,24 @@
+Analytics
```

Comment:
Stole this from [something bill wrote for finder frontend](https://github.com/alphagov/finder-frontend/blob/master/docs/analytics.md).
Seemed relvant